ui/components/src/ui-components/propertygrid/component/VirtualizedPropertyGrid.tsx

```diff
@@ -33,14 +34,18 @@ import { FlatItemNestedBorderWrapper } from "./FlatItemNestedBorderWrapper";
 import { PropertyCategoryBlock } from "./PropertyCategoryBlock";
 import { CommonPropertyGridProps, PropertyGridCommons } from "./PropertyGridCommons";
 import { PropertyGridEventsRelatedPropsSupplier } from "./PropertyGridEventsRelatedPropsSupplier";
+import { assert } from "@bentley/bentleyjs-core";
 
 /** Properties for [[VirtualizedPropertyGrid]] React component
  * @alpha
  */
 export interface VirtualizedPropertyGridProps extends CommonPropertyGridProps {
   model: IPropertyGridModel;
   eventHandler: IPropertyGridEventHandler;
-  highlight?: HighlightingComponentProps & { filteredTypes?: FilteredType[] };
+  highlight?: HighlightingComponentProps & {
+    filteredTypes?: FilteredType[];
+  };
+  propertyCategoryRendererManager?: PropertyCategoryRendererManager;
```

Comment:
Should be `@alpha`?

Reply:
Thanks, added in 190c6ba68b154d50acb35e81494009364051dab5.